Gems/ScriptCanvas/Code/Include/ScriptCanvas/Debugger/ClientTransceiver.cpp

```diff
@@ -435,13 +446,17 @@ namespace ScriptCanvas
 
         void ClientTransceiver::OnSystemTick()
         {
-            AzFramework::TargetManager::Bus::Broadcast(&AzFramework::TargetManager::SendTmMessage, m_currentTarget, Message::AddTargetsRequest(m_addCache));
-            m_connectionState.Merge(m_addCache);
-            m_addCache.Clear();
+            AzFramework::IRemoteTools* remoteTools = RemoteToolsInterface::Get();
+            if (remoteTools)
```

Comment:
minor: init remoteTools variable inside the if. 
if (AzFramework::IRemoteTools* remoteTools = RemoteToolsInterface::Get())